src/frontend/apps/lti_site/components/App/AppContentLoader/index.tsx

```diff
@@ -89,28 +91,34 @@ const AppContentLoader = () => {
     <QueryClientProvider client={queryClient}>
       <ReactQueryDevtools />
       <RawIntlProvider value={intl}>
-        <Grommet theme={theme} style={{ height: '100%' }}>
-          <Toaster
-            toastOptions={{
-              duration: 5000,
-              success: {
-                style: {
-                  background: colors['status-ok'],
+        <ErrorBoundary
```

Comment:
I would rather wrap Grommet context around ErrorBoundary to be able to use the theme in BoundaryScreenError component